mars_troughs/data/Obliquity_new.txt

```diff
@@ -0,0 +1,1002 @@
+Obliquity (units?) Time(yrs)
```

Comment:
I'm pretty sure the units of obliquity are degrees. You should double check with @ambramson on this though and also what the times are. @ambramson made the original obliquity text file.